ispyb/sp/mxacquisition.py

```diff
@@ -84,6 +84,14 @@ def upsert_dcg_grid(self, values):
     '''Insert or update the grid info associated with a data collection group'''
     return self.get_connection().call_sp_write('upsert_dcg_grid', values)
 
+  def get_dcg_grid(self, dcgid):
```

Comment:
I'd suggest using 'retrieve' as the verb and naming it 'retrieve_dcg_grid'. There are some inconsistencies in method names between this file and acquisition.py already ...!